src/lib/layer-manager.js

```diff
@@ -138,6 +167,42 @@ export default class LayerManager {
     return this;
   }
 
+  addEventListeners(eventNames) {
```

Comment:
> Hm...so then EventManager or an equivalent would be passed into LayerManager?

I had a different mode in mind. The external event handling would be independent and just call `layerManager.onClick` and `layerManager.onPointerMove` when events occurred to get help with picking and delivering the resulting infos to the deck.gl callback system in the right way.